sdk/eventgrid/Azure.Messaging.EventGrid/tests/EventGridEventTests.cs

```diff
@@ -140,7 +140,7 @@ public async Task RespectsPortFromUriSendingEventGridEvents()
 
         private static List<EventGridEvent> DeserializeRequest(Request request)
         {
-            var content = request.Content as Utf8JsonRequestContent;
+            var content = request.Content;
             var stream = new MemoryStream();
```

Comment:
```suggestion
            using var stream = new MemoryStream();
```

Reply:
not necessary as it is a memory stream.

Reply:
if disposable it is,  dispose it, you should.